addons/project/views/project_views.xml

```diff
@@ -1371,7 +1371,7 @@
                                             <field name="project_id" options="{'no_open': True}"/>
                                         </span>
                                         <span t-if="record.partner_id.value" style="display: block; margin-top: 4px;">
-                                            <field name="partner_id"/>
+                                            <field name="partner_id" class="text-truncate d-block"/>
```

Comment:
There is no longer `t-att-title`, is it expected? Do we manage that in the field?

Reply:
Oops! I forgot to add `t-attf-title` here. I will add this.